src/makefile

```
@@ -1,8 +1,14 @@
-libAzStorage.so: AzStorage.o
+local:
+	$(CC) `curl-config --cflags` -O3 -fopenmp -fPIC -c -o AzStorage.o AzStorage.c
+	$(CC) -shared -fopenmp -o libAzStorage.so AzStorage.o `curl-config --libs`
+
+yggdrasil: libAzStorage.${dlext}
+
+libAzStorage.${dlext}: AzStorage.o
```

Variables in Makefiles are dereferenced with `$()`, not `${}`
```suggestion
yggdrasil: libAzStorage.$(dlext)

libAzStorage.$(dlext): AzStorage.o
```